jquery.time-to.js

```diff
@@ -401,6 +410,12 @@
         $li.eq(1).html(n);
         n += step;
 
+        if (typeof this.data('step') === 'function') {
+            if (this.data('tickCount') % this.data('stepCount') === 0) {
+                this.data('step')();
```

Comment:
When `tickCount` reach MAX_SAFE_INTEGER further increasing won't change a value.
So I offer to reset this variable each time.